src/auth/auth.proto

```diff
@@ -79,13 +79,6 @@ message TokenInfo {
   // Subject (i.e. Pachyderm account) that a given token authorizes.
   // See the note at the top of the doc for an explanation of subject structure.
   string subject = 1;
-
-  enum TokenSource {
-    INVALID = 0;
-    AUTHENTICATE = 1; // returned by Authenticate()--non-revokeable
-    GET_TOKEN = 2;  // returned by GetToken()--revokeable.
-  }
-  TokenSource source = 2;
```

Comment:
There's only one way to get each token type, and all tokens are revocable, so I eliminated this concept.